code-studio/src/js/header.js

```diff
@@ -77,12 +77,17 @@ header.build = function (stageData, progressData, currentLevelId, scriptName, pu
   var isHeaderPopupVisible = false;
 
   function showHeaderPopup(target) {
-    sizeHeaderPopupToViewport();
-    $('.header_popup').show();
-    $('.header_popup_link_glyph').html('&#x25B2;');
-    $('.header_popup_link_text').text(dashboard.i18n.t('less'));
-    $(document).on('click', hideHeaderPopup);
-    lazyLoadPopup($(target).closest('.header_trophy_link').length > 0);
+    if ($(target).closest('.header_trophy_link').length > 0) {
+      // Only the 20-hour course has trophies.
+      location.href = '/s/20-hour#trophies';
```

Comment:
Yes, but why have that link have anything to do with `showHeaderPopup` anymore? Why not just make it an actual link that links to `/s/20-hour#trophies/`?